src/hieroglyph/builder.py

```diff
@@ -132,7 +132,7 @@ def post_process_images(self, doctree):
                 node['candidates'] = ('*',)
 
             # fix up images with absolute paths
-            if node['uri'].startswith(self.outdir):
+            if node['uri'].startswith(str(self.outdir)):
                 node['uri'] = '/'.join(
                     relative_base + [
                         node['uri'][len(self.outdir) + 1:]
```

Comment:
This one can fail too:

```suggestion
                        node['uri'][len(str(self.outdir)) + 1:]
```

There is a third use in `copy_static_files` below but that should be fine:

```python
>>> os.path.join('a', Path('b'), 'c')
'a/b/c'
```

Reply:
True, thanks.

By the way, of course all my PRs are available in https://github.com/lpirl/hieroglyph if that is of any interest to anyone.